presto-orc/src/main/java/com/facebook/presto/orc/CachingStripeMetadataSource.java

```diff
@@ -62,12 +66,15 @@ public Slice getStripeFooterSlice(OrcDataSource orcDataSource, StripeId stripeId
     }
 
     @Override
-    public Map<StreamId, OrcDataSourceInput> getInputs(OrcDataSource orcDataSource, StripeId stripeId, Map<StreamId, DiskRange> diskRanges)
+    public Map<StreamId, OrcDataSourceInput> getInputs(OrcDataSource orcDataSource, StripeId stripeId, Map<StreamId, DiskRange> diskRanges, HiveFileContext hiveFileContext)
             throws IOException
     {
         //
         // Note: this code does not use the Java 8 stream APIs to avoid any extra object allocation
         //
+        if (!hiveFileContext.isCacheable()) {
```

Comment:
Move this above the comment. The "Note" comment is for the ImmutableMap.builder code block.